### ISSUE TITLE
[bug reporter] [ios] 'done' button for dismissing keyboard only appears for the first time

### ISSUE THREAD

**Issue by cindyip** [USThing/USThing_Master#206]
### Expected behavior
'done' button appears and replaces the 'send' button whenever tapped on the typing space
### Actual behavior
'done' button only appears the first time when user tapped on the typing space
### Steps to reproduce
1. bug reporter
2. tap typing space
3. click 'done' to close keyboard
4. tap typing space again / type anything
### Version
ios 4.20 (34)

**Comment by tsznokwong** [comment]
@cindyip I don't see the issue, please check again?

**Comment by cindyip** [comment]
@tsznokwong still have... can't close the keyboard for the 2nd time

**Comment by tsznokwong** [comment]
@cindyip may I get your iOS version and type of device so I can check?

**Comment by cindyip** [comment]
@tsznokwong  sure 
iOS 10.2.1
iPhone 6
OS language: Chinese
